### ISSUE TITLE
output in the run_cojo rule

### ISSUE THREAD

**Issue by gmauro** [ht-diva/pqtl_pipeline_finemap#7]
output section of the run_cojo rule doesn't have all the output produced by the R script.
They should be added.
https://github.com/ht-diva/pqtl_pipeline_finemap/blob/092bc88673418139043ac383bd752ccc06e29421/workflow/Snakefile#L67

**Comment by dariushghasemi** [comment]
We firstly tested **Locus Breaker (LB)** R scripts for example meta-analysis GWAS results of the proteins and we deployed it in Snakemake (SMK) in mid April. 

We started to deploy **COJO conditional analysis** in the 3rd-4th week of April. As some steps like parallelization of jobs were implemented using built-in strength of NF, one we were transferring the fine-mapping analysis (step 4) into SMK, we had to loop the analysis over the index variants (P<5e-8) identified in LB analysis, in parallel for each protein sequence. We iterated it using R script `s04_cojo_finemapping.R` inside 'rule run_cojo'. Therefore, we read TSV file containing the index variants and iterating the R script only for those variants. So that if there is no genome-wide significant variants for a protein sequence, rule 'run_cojo' would fail for that protein.

Outputs of `rule run_cojo` for fine-mapping analysis include:
- list of independent variants resulted from GCTA cojo-slct (TSV/CSV)
- conditional dataset for each independent signal resulted from GCTA cojo-cond (RDS)
- Fine-mapping results using _coloc::coloc.ABF_ function, containing values such as l-ABF, posterior probabilities (PPI) for each variant (RDS)
- colocalization info table containing credible set variants (with cumulative PPI > 0.99) for each independent variant
- regional association plots

Outputs are saved here: `/scratch/dariush.ghasemi/pqtl_pipeline_finemap/results/*/cojo/<seqid>` (Thu, 23:45, 25-Apr-24).
____________________________________________
The first two steps of the original workflow are not needed for the pQTLs pipeline. Therefore, it is better to start with the step 3, locus breaker, directly using protein GWAS summary stats from REGENIE which is already aligned (Fri, 11:58, 26-Apr-24).

Some commits helping to trace the development of the pipeline:
- Locus breaker is incorporated into the pipeline. Time to do so with COJO execution (Sat, 03:45, 27-Apr-24).
- Testing the pipeline on the meta-analysis results modified by GWASLab (Sat, 16:46, 11-May-24).
- Verifying the results using GLM model (Thu, 03:12, 18 to 20-Jun-23).
- Lunching the locus breaker and COJO on the latest meta-analysis results (Mon, 23:55, 24-Jun-24)
- Running locus definition and COJO using updated PGEN/BED genotypes got finished (Tue, 19:30, 10-Jul-24).
